New version: PowerSimulationsDynamics v0.14.0

JuliaRegistrator opened the pull request:
- Registering package: PowerSimulationsDynamics
- Repository: https://github.com/NREL-Sienna/PowerSimulationsDynamics.jl
- Created by: @jd-lara
- Version: v0.14.0
- Commit: 3f9dede8867a4a184b1509df64f2de4481ebbab3
- Reviewed by: @jd-lara
- Reference: https://github.com/NREL-Sienna/PowerSimulationsDynamics.jl/commit/3f9dede8867a4a184b1509df64f2de4481ebbab3#commitcomment-138483739
- Description: Julia package to run Dynamic Power System simulations. Part of the Scalable Integrated Infrastructure Planning Initiative at the National Renewable Energy Lab.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd63f5ff04afab822ff987f727e8a2abc37080b2f920ec4eab85ac5342fd2a90f6c916815a7998666c52685de83af69d4bf9c4b81522d0bcede7680ffaaeefebda07119b6128bd75f12c5fa14867f4a33b1e7619b48c0ac37e75a603a47161bf9582b0ca0e15f748cf7f70a4838d1c6944327ea842b8f45794076e4e4169dedc7e2353d3930e4903c16511a41af167fd0e76aaf969472ac51f432a3137cf95692f990c92dc2dc1d5c523ad6865ccc49784a3b27bcb23d8e3ff914f4e11dd14e7016 -->

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@jd-lara Can you re-trigger Registrator? This will fix the merge conflict.

JuliaRegistrator committed:
New version: PowerSimulationsDynamics v0.14.0

UUID: 398b2ede-47ed-4edc-b52e-69e4a48b4336
Repo: https://github.com/NREL-Sienna/PowerSimulationsDynamics.jl.git
Tree: 89da36da74126b53d6082899b80e968553a16417

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
